### ISSUE TITLE
Showcase example

### ISSUE THREAD

**Issue by ajs6f** [linked-art/linked.art#220]
As per discussion on [this call](https://docs.google.com/document/d/1vMeVtaXlFmorNnumzeQp5lkrAunZPZzklnzHRGPNwGI) we are going to assemble a "showcase" example for use on a call with curators and collections managers.

**Comment by edgartdata** [comment]
Damien Hirst's _In and Out of Love (Butterfly Paintings and Ashtrays)_ consists of 8 paintings, household paint on canvas with butterflies; 4 boxes; 1 table, formica top on steel base; and 4 glass ashtrays filled with cigarette butts. 
https://collections.britishart.yale.edu/vufind/Record/1669159 

![image](https://user-images.githubusercontent.com/4093644/55093458-ca39b800-508a-11e9-8cde-9408e69ed9d7.png) 
![image](https://user-images.githubusercontent.com/4093644/55093618-184ebb80-508b-11e9-8742-4d91f1f4ecd9.png)

It does not look like a particular order must be respected for the paintings? The Tate Modern had a different sequence for the paintings in 2012: http://www.damienhirst.com/in-and-out-of-love-butterfly
![image](https://user-images.githubusercontent.com/4093644/55094123-26510c00-508c-11e9-8418-1e1db862e269.png)

Regardless, it might be useful to document the particulars of each new installation for the piece.

Oh, and it is itself only half of the original overall installation (see curator's comment in https://collections.britishart.yale.edu/vufind/Record/1669159).

Bonus question: would we want to model the piece down to the individual cigarette butts?

**Comment by ajs6f** [comment]
@workergnome @SamiNorling @beaudet @danieltbrennan (not sure of Kevin Page's Github ID)

This is an issue to which we can attach our ideas or items for use in a showcase example. I'm hoping y'all have ideas about what would be interesting/exciting to curatorial staff, because I live in a central office here at SI, not any of the museums that might be expected to hold good examples, and I am no domain expert!

**Comment by ajs6f** [comment]
Wow, @edgartdata, that was fast, thank you!

**Comment by ajs6f** [comment]
@edgartdata, from conversation this morning, I was thinking that we want an example that shows connection across institutions (federation, as @workergnome put it). I honestly don't know enough about [Damien Hirst](https://github.com/janephilipps/spot-painting) (or contemporary art, generally) to know whether we'll have a good chance at finding those connections. What do you think?

**Comment by SamiNorling** [comment]
I agree that focusing in on the parts of the Linked Art model that offer the most interesting opportunities for highlighting links between institutions' collections, creators, exhibitions, etc. would serve us the best in demonstrating the utility and value of linked data for art collections. Though we discussed it in detail today, I don't know if complicated partitioned artworks would fit this bill.

What first comes to my mind as something that could be explored (and that I know our collections managers and curators would be intrigued by) are:

- Common collectors/donors across institutions, so modeling a few objects that share a common provenance point of some kind;
- Exhibitions (though development of this part of the model won't take place for a while) - which pieces have traveled where and when to be exhibited; trends of artists represented in exhibitions across multiple institutions' exhibition history;
- Relationships between artists as a way to evaluate and interpret works - reconciliation with ULAN in modeling creators would make this possible, since ULAN documents relationships between individuals/organizations (e.g., student of) - this could open up an opportunity to get a curator involved, who could speak to a specific individual, school, or artist group and their web of influence

Those three ideas are the first that come to mind, but there's many more opportunities for exploration across collections that the Linked Art model will open up in the future...

**Comment by edgartdata** [comment]
I would think that the issues raised by contemporary art installations with many parts would be both relevant to several institutions and bears some similarities to more traditional use cases, such as the panels and predella of a triptych being detached and/or dispersed across several locations at one museum or across several institutions.
But I will keep thinking of other use cases.

**Comment by ajs6f** [comment]
@edgartdata They very well may be; I'm coming to think that we eventually want a _suite_ of nice examples for various audiences/purposes.

@SamiNorling I agree that those commonalities would make for good cross-institution linking. With respect to your third idea (and pardoning my ignorance of the domain), do you know of institutions that have accomplished reconciliation against ULAN, or is that something we could just do "manually" for an example?

**Comment by SamiNorling** [comment]
@edgartdata Your mention of detached and dispersed groups of items is definitely a good lead - hadn't thought of that aspect of partitioned pieces.

It brings to mind the separation of the frescoes from the Hermitage of San Baudelio de Berlanga. We have [two of the frescoes in our collection](http://collection.imamuseum.org/results.html?query=San%20Baudelio), and I know of other frescoes that are in the collections of the Cincinnati Art Museum, The Met, BMFA, and Museo del Prado.

**Comment by SamiNorling** [comment]
@ajs6f The partners of the American Art Collaborative had some ULAN reconciliation completed, and we still have that data linking our artist identifiers with ULAN identifiers, where available. Multiple AAC partners either already had identifiers in their CMS or added them once they had the data back. We also document this in our CMS to some extent, though I'm hoping to do a lot more. I use OpenRefine in my bulk reconciliation efforts.

For this demonstration, I think we could definitely handle small scale reconciliation if it's not already completed. The local example I have used for the benefits of linked data and taking the time to reconcile with external authorities is William Merritt Chase, who was the teacher of Georgia O'Keeffe - both artists that are represented in our collection, but that relationship is not catalogued in our CMS (and really, it wouldn't be practical for us to catalogue that detailed of information at any sort of scale). So reconciling with ULAN across all of our creators, and publishing that data as linked data then adds our collections to the full web of relationships that ULAN captures. That example has gotten a positive response internally as an argument for linked data and/or reconciliation work, depending on what I'm advocating for at the time.

**Comment by danieltbrennan** [comment]
Still thinking about this a bit but I really like that frescoes example @SamiNorling. In that vein we have a substantial number of objects (mosaics, fragments, etc) that were excavated from Antioch and then dispersed to different institutions.

https://artmuseum.princeton.edu/collections/objects/29551
https://en.wikipedia.org/wiki/Antioch_mosaics

Potentially an interesting case since (at least for ours, but I would imagine for the other institutions as well) their history and movements are pretty well-documented.

**Comment by ajs6f** [comment]
@danieltbrennan Do you think we could ask those institutions to collaborate with us, or were you thinking that we would ourselves use whatever information we could get from each  and build up a Linked.Art description ourselves?

**Comment by danieltbrennan** [comment]
@ajs6f good question, I was more thinking the latter - that's a lot to ask from institutions that may not have the bandwidth for it (although some may be game). 

Something I'd float as reasonable only because I know through past experience/projects that never came to fruition that there is pretty ample documentation around that set of works.

**Comment by ajs6f** [comment]
@danieltbrennan That makes sense. I would be interested in working on that. We could try coming up with a skeletal set of interlinked descriptions then enrich if we decide it's a good road to go down.

@SamiNorling Do you think that either the frescoes from the Hermitage of San Baudelio de Berlanga or @danieltbrennan 's Antioch examples would give us an opportunity to show the "ULAN effect" wherein reconciling some metadata and connecting it to some "external" graph offers actual new knowledge?

**Comment by azaroth42** [comment]
Here our Antioch mosaic:
http://www.getty.edu/art/collection/objects/6539/unknown-maker-mosaic-floor-with-animals-7-roman-about-ad-400/ 

:)

**Comment by ajs6f** [comment]
Excellent, so the Antioch example might already have two Linked.Art institutions! I'll offer this idea for review on the next call, but I already think the Antioch example is promising.

@edgartdata would you be interested in working up the Damien Hirst example with me?

**Comment by SamiNorling** [comment]
@ajs6f Not sure either of these object groups would show the benefits of reconciliation and linked data that I mentioned, but I think they are both strong examples to demonstrate the provenance portions of the LA model.

I think what I outlined with ULAN reconciliation could be an interesting demonstration of the value of linked data. It's not as model-heavy as the possibilities we've discussed, since it would place more of an emphasis on the importance of reconciliation with controlled terms/entities when creating and publishing linked data. Still, an important piece to demonstrate, and could provide a compelling argument for linked data from an art historical perspective. Just thinking of our curatorial staff, I think they'd see a lot of value in the sort of inferences that can be made about our collection when we just link to other data with a much broader scope than our current cataloging practices.

I'll look into this more, but am also happy to contribute the frescoes data and try to work with other institutions' data that may have one or multiple of the frescoes as well. I imagine that among our group, we'd have contacts at other holding institutions (Cincinnati, BMFA, The Met, etc.).

**Comment by ajs6f** [comment]
Okay, I'm hearing (at least) three good examples, of different kinds, each showing a different aspect of our project:

1. The Antioch mosaics.
    - the power of cross-institution linking, and could provide a platform for an interesting example application
2. The Damien Hirst example
    - the power of a sophisticated model to describe really multifarious or weird things that are really difficult to describe otherwise, all within one institution
3. A "ULAN example"
    - the connections that arise when Linked Data is in use (regardless of model) and institutions make the effort to use good identifiers

@SamiNorling, do you have a concrete example in mind for 3, or would you rather pick an interesting artist (perhaps someone, as you mentioned, for whom we could get some curatorial interest) and see what we all have when we put our resources together? Of course we don't have to actually reconcile our data against ULAN to do this; we can "can" an example by choosing an artist and manually selecting records for use therewith.

**Comment by ewg118** [comment]
Kerameikos.org is going to transition to using Linked Art compliant CRM, or at least harvest Linked Art JSON-LD and perform a light transformation to include several properties that Greek pottery scholars rely on for query.

**Comment by zeroexp** [comment]
Hello - I have been on the call for the past weeks. I have a tool for adding linked data triples to images and then sharing and publishing that data and those triples in rdfa and in JSON-LD. It is an application, designed such that the subject URI can be any valid URI -- in other words, I did not copy the image shown here from the V&A museum, but instead simply used the URI of their image as the subject URI - you can note this in the triples.  If you have an extension in your browser (like the Data Sniffer) tool, OR you can look at the source of this file and you can see the linked data triples I added to this example - which was one of the cabinets used as an example by the V&A museum: http://imgsnp.co/jy5e0 --- in this *particular* case, we are using a lightweight vocabulary of terms to relate the keyword to the image, however each keyword can come from the CRM or the AAT or any other vocabulary.  This particular example also uses RDFa in the source. Up until recently, we had been including JSON-LD only for the images when they are published on a web page. However, I will be adding JSON-LD to the source of every image in the next few weeks.  I have oIther examples I can provide as well and I can provide the login such that people can experiment with adding triples to the images of their choice.  In this way - the app functions as an experimental platform to add the triples to see how linked data actually works on the web.

**Comment by zeroexp** [comment]
here is another example from the met open access images: http://imgsnp.co/k5lq4 --- in which I have added the man-made object term using the CRM and used the met accession number as an identifier.  This is an experimental platform and I would love the opportunity to demo the work as it directly relates to the idea of attempting to build a shared linked art model.

**Comment by zeroexp** [comment]
![image_triple_editor](https://user-images.githubusercontent.com/45359050/55839491-0989ef00-5af6-11e9-96dc-a6ba1a6bad42.jpg)

**Comment by zeroexp** [comment]
![selection_of_crm_term](https://user-images.githubusercontent.com/45359050/55839556-5077e480-5af6-11e9-81f5-bedf59a24316.jpg)
This image shows the selection of a term from the CRM ontology which is just one of the datasets available for looking up linked data terms - again, with relative ease.

**Comment by zeroexp** [comment]
![dataset_subscription](https://user-images.githubusercontent.com/45359050/55839603-74d3c100-5af6-11e9-9458-d94aaae5cb36.jpg)
This shows the subscription of various linked datasets. Addtionally, there is a way to build a custom dataset.

**Comment by zeroexp** [comment]
![data_viewer](https://user-images.githubusercontent.com/45359050/55839663-aba9d700-5af6-11e9-992c-b5a4fb87244b.jpg)
this example shows the use of a completely independent extension made by Open Link Software that shows how the applied linked data metadata can be followed to the AAT or Wikidata or any other chosen linked dataset.

**Comment by zeroexp** [comment]
![data_viewer2](https://user-images.githubusercontent.com/45359050/55839722-e4e24700-5af6-11e9-830b-5805b6072ce1.jpg)
and finally - here is a different example from the V&A Museum - the example of the tuning forks in the case.  NOTE --- additionally, we have all inference implemented in ImageSnippets for AAT, DBpedia and Wikidata. Which means you can search for superclasses -- such as 'sound device' in the interface and see how the actual class inference works with linked data.

**Comment by zeroexp** [comment]
![json-ld_syntax](https://user-images.githubusercontent.com/45359050/55839865-6e921480-5af7-11e9-9eaa-c0abce9cc0b9.jpg)
and finally - this is an example of an image from Wikimedia which shows the same data expressed as json-ld

**Comment by zeroexp** [comment]
Ethan - great example!

**Comment by danieltbrennan** [comment]
Dropping a note here mid-discussion on other possibilities for the artist-network (ULAN) example, I'm reminded that 20th century photography is particularly rich in well-documented figures/relationships -  so that might be another area in which to look for examples. 

I'm reminded of this (pre-linked data) project: 

https://www.moma.org/interactives/objectphoto/#connections?dateBegin=1900&dateEnd=1950

**Comment by ajs6f** [comment]
Now I'm idly wondering if anyone has done anything to visualize the enormous number of relationships in ULAN… It might be possible to find a particularly good "target" by finding someone with really interesting connections. Hm…

**Comment by ewg118** [comment]
I don't believe relationships of this nature are even encoded in the ULAN, which is why there are various stand-alone SNA projects connected to ULAN entities, but these relationships were created outside of the ULAN's own LOD structure. There's more relationship modeling at a large scale in https://snaccooperative.org/

**Comment by ajs6f** [comment]
Hi, @ewg118, we can say that such links are encoded in ULAN at least some of the time. Consider [Georgia O'Keefe's record](https://www.getty.edu/vow/ULANFullDisplay?find=o%27keefe&role=&nation=&prev_page=1&subjectid=500018666), which shows her (typed) relationships to Alfred Stieglitz, Charles Demuth, William Merritt Chase and others. I don't know how widespread such links are, but we can certainly find out. @azaroth42, @workergnome, others… thoughts about how normal it is for ULAN to record person-to-person relationships?

SNAC is certainly something we can and should look at to find links, but I'm not sure we would do well to draw our "showcase" examples from it. I wouldn't say it's necessarily something that curators or collection managers in art museums are going to be familiar with.

**Comment by ewg118** [comment]
@ajs6f You're right about the relationships. I've never come across it before because it seems to be unusual. I put together a query that would extract all of the people that have relationships with other people. There are 1800 total relationships expressed between over 400 people. You can try the SPARQL query here: https://gist.github.com/ewg118/2710396a1a7136a5d4e59a07d18da270

SNAC probably isn't a good showcase example yet because there aren't a lot of cultural heritage materials aggregated into the system. However, there are some art museums working on inserting updated biographical or relationship information into it. I think MOMA is, and the Fitzwilliam is exploring this as well. But SNAC is one example that is putting a lot of resources into mapping relationships between entities at a larger scale than most of the DH/prosopography projects operating within a more narrowly-defined academic area.

**Comment by SamiNorling** [comment]
@ajs6f @ewg118 The results of that query definitely don't include all relationships between two person entities in ULAN. Seems to be the results are only individuals that are rulers of some kind. Something about the path to the prefLabel in English was limiting the return. I simplified the ?personName and ?otherPersonName to populate with skos:prefLabel (no filter for language) and the total number of triples is 118,818 which could be halved for a more accurate total (59,409) because it looks like relationships are expressed in two ways: generally with a simple skos:related statement, as well as with a more descriptive predicate (e.g., gvp:ulan1102_student_of).

But just querying William Merritt Chase (keeping with the example), there are 83 unique relationships expressed in ULAN.

`SELECT ?personName ?rel ?otherPersonName ?otherPerson WHERE {
  ulan:500115356 ?rel ?otherPerson ;
         skos:prefLabel ?personName .
  ?rel rdfs:subPropertyOf skos:related .
  ?otherPerson a gvp:PersonConcept;
         skos:prefLabel ?otherPersonName .
  }`

**Comment by ewg118** [comment]
Interesting. There's a lot of inconsistency in the way labels and definitions are modeled in the Getty's underlying RDF. I guess they established a skos:prefLabel through inferencing. It's not explicit in their underlying data. Labels could be located in an object URI defined by the property `xl:prefLabel` or `gvp:prefLabelGVP`, and if you go to the URI at the end of one of these properties, the literal label might be encapsulated in one of several possible properties.

**Comment by workergnome** [comment]
The person-to-person relationships are pretty rich, actually, but @ewg118 is right that the current vocabs LOD relies really heavily on inferencing.   

http://vocab.getty.edu/ontology has up a bunch of that information, with more documentation at http://vocab.getty.edu/doc/#Relationships_Table.  You can also look at the sample queries at http://vocab.getty.edu/doc/queries/#Associative_Relations_of_Agent -- for example, for Duerer:

```
select * {
  ulan:500115493 ?rel ?x.
  ?rel sesame:directSubPropertyOf skos:related.
  ?x gvp:prefLabelGVP/xl:literalForm ?name.
  ?x foaf:focus/gvp:biographyPreferred/schema:description ?bio.
  optional {[
    rdf:subject ulan:500115493;
    rdf:predicate ?rel;
    rdf:object ?x;
    rdfs:comment ?comment]}}
```

**Comment by ajs6f** [comment]
If I understand correctly, we have access to the rich "social network" recorded in ULAN, although we may have to wax a bit clever in how we consume it. Sounds fine to me for the moment; we are building examples and not production systems, for the moment.

Who will make a good example? That's way, way out of my game. All I could say is that we want someone who will interest curators and who has a rich enough set of relationships recorded about them and a rich enough oeuvre as distributed amongst our institutions that we can create something interesting with their data.

**Comment by workergnome** [comment]
When we did the AAC browse app, the single most well-represented individual was Georgia O'Keefe, who also has rich relationships in ULAN.

**Comment by ajs6f** [comment]
That's a big point for her. Certainly she is also a painter of wide public renown, although I'm not quite sure how important that would be right now to us.
Is there any way we can get a quick sense of who amongst us has art by her in our collections? We have at least [some stuff](http://collections.si.edu/search/results.htm?q=georgia+o%27keefe), although I would have to get help from curators here to be more useful than that.

**Comment by edgartdata** [comment]
It will come to no surprise that the YCBA doesn't. 
How about starting from an institution, such as the Royal Academy or others, that has many members over a long period of time? Using such a membership organization like that would be more likely to produce results across many of our collections.

**Comment by ajs6f** [comment]
@edgartdata That's a great idea! What do you think would be a good example institution-- the Royal Academy?

**Comment by edgartdata** [comment]
RA could be a good one. Chronicle250 has a handy index of 250 years of its exhibition catalogs: https://chronicle250.com/index/exhibitors 
Maybe the French Royal Academy of Painting and Sculpture? Here is a partial list of its members: https://en.wikipedia.org/wiki/List_of_members_of_the_Acad%C3%A9mie_Royale_de_Peinture_et_de_Sculpture

**Comment by SamiNorling** [comment]
Asking how many of our institutions have Georgia O'Keeffe works won't give us a good measure of whether this example is worth pursuing, since ultimately we're casting a net for the "social network", starting with O'Keeffe (or whoever we go with) as the hub.

The evaluation should be about degrees of separation - which works in our collections have creators that can be connected to O'Keeffe in, for example, three or less degrees of separation?

From her ULAN record, her first degree network includes:
- F. Luis Mora
- Kenyon Cox
- William Merritt Chase
- Charles Demuth
- Ruth Carter Stevenson
- Ida O'Keeffe
- Alfred Stieglitz

The second degree network would be an even larger list. This full picture is what we should be checking our collections for and seeing who can contribute data.

This approach will give us the same starting point that picking an institution would - a larger list of artists to consider - but with more varied connections between the links, as opposed to just all "members of". Though I would assume that starting with an organization would end up giving us artists who have a variety of relationship between each other, just from having the common organization to have connected them.

**Comment by zeroexp** [comment]
I love this idea! For what it's worth and for the record - I don't intend to turn all of my interactions in this group into an ImageSnippets infomercial - it's just happens that it is  a framework I built for doing exactly what we are doing in a way - so it's like my child. I am also an artist and a pure research scientist far more interested in pure research in knowledge representation than I am in profit.

 I think the Georgia O'Keefe example is an excellent because of all of the interrelationships with other artists, how do you intend to show those relationships with the linked art model?  

Another interesting suggestion might be Matisse and Picasso ---big examples, yes - but not insurmountable, particularly where there are specific works known to have been influenced by the other. 

Currently, I have a convention I have been using to related works to each other or to other artists; but I am curious if there is already a convention that some of you use for inter-linking those relationships.

**Comment by ajs6f** [comment]
@SamiNorling makes a really good point-- it's about the relationship between artist and artist at least as much as the relationship between artist and art.

I guess one way to ask the question would be: do we want to try to _present_ an example based on a single artist or on some organization? Which one would be easier to explain? Which one would be more attractive? (Maybe both to different audiences?)

**Comment by azaroth42** [comment]
At the *current stage* of our discussions, where the emphasis has been on describing the object, I'd prefer to stick with person / object, rather than person / person.  For example, we have no vocabulary (nor agreement on ontology) for social network style relationships that would be needed to fully describe the relationships between people.

**Comment by ajs6f** [comment]
I'm not at all comfortable with that restriction, @azaroth42. For the audiences I have to engage here at SI, the difference between a vocabulary we own and some other vocabulary is totally uninteresting. They want to see _why_ we should invest in Liked.Art at all, and I'm not sure in the absence of the ability to link beyond person / object that I can tell them why they should. We might be running into a distinction between examples that are useful for education and those that are useful for evangelism. 

Can we get an agenda item for this on this week's call?

**Comment by SamiNorling** [comment]
Note: I'm posting this comment before reading additional comments by @ajs6f and @azaroth42 posted as I was writing this...

@ajs6f I think both approaches could lead to a really interesting example - we're just at that hard part of picking one and going with it to see where it leads us. I think there'd be great interest in either.

To think through this a bit...an outline for the demonstration may include (but not be limited to):

1. Provide a simplified, brief overview of linked data principles (this is necessary IMO, based on my experience in demonstrating specific linked data projects - inevitably there are lots of higher level questions) - Getty ULAN data artist relationships has always been my go-to example for this, which is why we've focused in on this during the discussions - it doesn't really serve the Linked Art demo, but can lay the foundation for understanding "why linked data?".

2. Ultimately, we want to move past the general linked data overview to the meat of our demonstration, which is the Linked Art model. Key here is that the model provides a standard structure for publishing linked data about art on the web, which can be implemented by anybody, to make standard what is often represented in very different ways across institutions.

3. The payoff/ROI: What opportunities for analysis/exploration does a shared model for linked art data open up? This is where we would use the data gathered for our particular example. This could be data about artworks produced by artists within a specific social network, if we want to directly build off the foundational linked data overview (assuming we provide that overview, but not assuming that it has to be the ULAN example). The other option we've discussed is for this artwork data to be the works of art produced by individuals associated with an organization.
Either way, we want to show some interesting things about the artworks when analyzed as whole, facilitated by the shared data standard.

**Comment by azaroth42** [comment]
Agenda wise - Certainly! 

Perhaps we should clarify the intent of the discussion - is it to make a generic pitch for linked data being valuable, or is it to demonstrate why institutions should care about linked.art in particular?  If the former, then any sort of distributed relationship pattern is fine, but my feeling is that pitch has been made over and over again, and people now want to see something tangible, not just be offered koolaid. If the latter, such that we can offer tangible, implementable solutions, then we shouldn't open ourselves up for the obvious criticism *at this stage* of "So how do you say that artist X is the grandmother of collector Y?"

**Comment by ajs6f** [comment]
Yes, that's the distinction I'm making between education and evangelism. At this point, I think at my institution there's basically no audience for education without some successful evangelism. No one wants to see Yet Another Interoperability Standard (TM) unless it offers clear benefits, and to be frank, I'm in that group too. If after all the work of AAC and all the additional work of smart folks at Getty, we can't actually offer an example of anything more sophisticated then a single artist and a single object…

I'm sympathetic to your point about how little we've done to understand interpersonal relationships. I just don't see why that would prevent us from presenting something that uses them, _in howsoever a speculative form_, to actually show why Linked.Art is interesting. To put it another way, to your point about opening ourselves up to criticism, how much longer can we wait to be able to say something as simple as "artist X was a student of artist Y"?

**Comment by SamiNorling** [comment]
Now jumping in to respond to the recent comments by @ajs6f and @azaroth42 ...

To clarify: I did not anticipate person-to-person relationships to be part of the Linked Art spec (at least not in Phase I). It seemed that ULAN/relationships between artists could serve as a good starting point for narrowing down the artworks for which we would want to gather Linked Art JSON-LD examples, ideally from multiple institutions. E.g. - knowing that [these artists] had [these relationships] (student/teacher/student/teacher network, for example) - what are some commonalities about their works? - Shared materials/styles/techniques? Geographic distribution? Length of production process? etc. etc.

**Comment by ajs6f** [comment]
@SamiNorling That's a really good point, and I should clarify my position; I don't want to see social relationships modeled in Linked.Art now. I _do_ want to _use_ them (expressed in any convenient form, no matter how unscholarly or inexact) to demonstrate why Linked.Art is interesting and useful. I don't have confidence that we can generate a really useful model without empirical feedback (doing hard examples) or without stealing as much as possible from extant work (reusing ULAN relationships).

**Comment by SamiNorling** [comment]
[ThreeDegreesOfOKeeffe.xlsx](https://github.com/linked-art/linked.art/files/3112376/ThreeDegreesOfOKeeffe.xlsx)

Attached is a spreadsheet with the output of a query to ULAN SPARQL endpoint to grab three degrees of relationships starting with Georgia O'Keeffe. If this is the example we go with, any artist listed here is fair game for any museum partner to generate sample Linked Art JSON-LD representing their artworks.

If we go this route, the "social network" provides the framework for our selection of artworks to model, as well as an easily-understood and (IMO universally interesting) example of linked data, should we decide a general LOD overview is necessary for our audiences before diving into the Linked Art model. Once we have a chunk of artwork data collected (and we could focus in on a specific set of artists if it would help, since the list right now is pretty large), then the main demonstration, along with the model, would focus on any interesting facts/trends/visualizations/etc. we can distill from consuming the data in bulk.

**Comment by SamiNorling** [comment]
Updated Three Degrees of O'Keeffe spreadsheet:
[ThreeDegreesOfOKeeffe_v2.xlsx](https://github.com/linked-art/linked.art/files/3112973/ThreeDegreesOfOKeeffe_v2.xlsx)

Clarified column labels re: IDs, and includes URI prefixes:
ULAN entities - http://vocab.getty.edu/ulan/
ULAN relationships ontology - http://vocab.getty.edu/ontology#

**Comment by azaroth42** [comment]
[This query](http://www.getty.edu/art/collection/search/?view=grid&query=YTozOntzOjU6InRpdGxlIjtzOjE2OiJnZW9yZ2lhIG8na2VlZmZlIjtzOjg6Im1ha2VyLmlkIjthOjE6e2k6MDtpOjE4MTU7fXM6NDoic29ydCI7czo0MzoiLXNvcnQuZGF0ZS5iZWdhbi55ZWFyLC1zb3J0LmRhdGUuZW5kZWQueWVhciI7fQ%3D%3D) results in 59 photographs by Stieglitz of Georgia O'Keeffe that we have in our photography collection.  *Warning: artistic nudity*

These are interesting, model wise, on two fronts -- the artist's relationship to the subject, and the role of O'Keeffe as being depicted, rather than being the artist.

**Comment by lizneely** [comment]
Hello! This is Liz Neely jumping in from the Georgia O’Keeffe Museum. As some of you know, we’ve been working on a project to deliver data from our collections as linked open data using Linked Art as a target model (learning from the AAC). The project includes data from our fine arts collection, our archives, a collection of the artist’s personal property (and the two historic homes we manage), and her personal libraries. We are doing this work with Design for Context and Charlie is doing the lifting with the data.
 
I am thrilled with the prospect that GO’K could be a showcase example and would love to support that in any way possible! I also think she is a great example (though I am a bit biased!)

One thing I wanted to bring up is thinking about the role of ULAN as a key glue, but perhaps more in connecting links than in limiting ourselves to what ULAN itself has entered – we’ve been thinking about this, and the 3 degrees of O’Keeffe that Sami pulled illustrates the point of why we need to use ULAN to get beyond ULAN data. 

For example, in the 3 degrees of O’Keeffe data using ULAN, Ansel Adams is listed as a 3rd degree connection to O’Keeffe. But, just looking at our collection data (and at the objects in the AAC), Adams photographed O'Keeffe as early as the 1930s and as late as the 1980s, they traveled to Yosemite together, he took installation photos of her exhibitions, they have both been in exhibitions together, there have been books about their friendship.  So, the data tell a different story than ULAN does. 

As a showcase to a subject expert, just using ULAN and showing Adams as a 3rd degree to O’Keeffe would quickly discredit the value we try to express. Additionally, every artist has important connections that aren’t limited to artists in ULAN (In our data (mind you, as a single artist museum we might be an outlier), we are almost evenly split between ULAN, LCNAF and local sources for authorities. Wikidata may help us deal with some of those local sources.)

Could it be that instead of the connections centering around ULAN, that instead ULAN  'learns' from the data of the linked data collections themselves and from the other authorities, which might tell a more nuanced story? (linked data FTW!)

I’ll be joining your call next week and would love to hear your thoughts.
Thanks – and happy Friday!

Liz

Liz Neely
Curator of Digital Experience
Georgia O’Keeffe Museum
Santa Fe, NM

**Comment by ajs6f** [comment]
@lizneely, thanks for joining in! I agree that relying solely on ULAN would give us a very sparse and unreal picture of the world. I think we gravitated towards it because it is a Getty product and because so many of our sites use it, but indeed to present a compelling example we would need more relationships than it can supply. Whether ULAN can "learn" from our LD work is an interesting question, but I suspect it has as much to do with how the ULAN team intends to manage that system as much as anything we do in Linked.Art. Getty folks, of course, would know much more than I.

Can you tell us a little more about the linked data you are currently producing? What vocabularies (aka join points) are you using for that?

**Comment by SamiNorling** [comment]
Hi @lizneely, glad to see you jumping in as well!

We investigated ULAN relationships pretty much just as a way to help us set a scope for collecting artwork data (since we started with the scope as ALL OF OUR COLLECTIONS), and also because it could provide an interesting intro to linked data example, if that ends up being needed. That potential intro example becomes even stronger if we can include the linked data that GOKM is publishing. One of the greatest benefits of publishing linked data is that, if you do it right (i.e., using/reconciling with official entities like ULAN records), it ultimately connects local data with much more rich, comprehensive, and potentially more authoritative data on the web (a.k.a., GOKM's linked data about O'Keeffe).

Sami

**Comment by zeroexp** [comment]
I'm curious - after looking ath the AAC work and this example page on Georgia O'Keefe: http://browse.americanartcollaborative.org/actor/ulan/500018666.html 
which shows relationships of O'Keefe to Stieglitz as well as teachers and colleagues as well as works pulled in from multiple collections.  Is this not something that already showcases the value of linked data in many of the ways described here? In what way is this showcase trying to go beyond something already seen here at this link at AAC?

**Comment by zeroexp** [comment]
> [This query](http://www.getty.edu/art/collection/search/?view=grid&query=YTozOntzOjU6InRpdGxlIjtzOjE2OiJnZW9yZ2lhIG8na2VlZmZlIjtzOjg6Im1ha2VyLmlkIjthOjE6e2k6MDtpOjE4MTU7fXM6NDoic29ydCI7czo0MzoiLXNvcnQuZGF0ZS5iZWdhbi55ZWFyLC1zb3J0LmRhdGUuZW5kZWQueWVhciI7fQ%3D%3D) results in 59 photographs by Stieglitz of Georgia O'Keeffe that we have in our photography collection. _Warning: artistic nudity_
> 
> These are interesting, model wise, on two fronts -- the artist's relationship to the subject, and the role of O'Keeffe as being depicted, rather than being the artist.
> 
> Further, [this object](http://www.getty.edu/art/collection/objects/99672/alfred-stieglitz-georgia-o'keeffe-a-portrait-american-june-1917/) depicts O'Keeffe standing in front of one of her watercolors, "Blue I." which was sold by Christies: https://www.christies.com/lotfinder/Lot/georgia-okeeffe-1887-1986-blue-i-5994689-details.aspx
> 
> And ... note the orientation of the painting in the background ... it's landscape. Christies has it depicted as portrait, which is then used in pretty much every other copy of the image I can find online. Given that she painted it in 1916, and the photograph was 1917, why would she have it oriented in a way that she wasn't intending it to be seen?

@azaroth42 ---- this is just an aside to this whole conversation because, I think this is a neat question. Wondering if @lizneely noticed this observation and has any guidance on the orientation of this painting as O'Keefe herself directed. Putting on my artist hat, I can think of many reasons why an artist might change the orientation of a painting to suit - in this case - the photograph being made because of the way the shape surrounded her head behind her. As an abstract artist, sometimes I am critically aware of what orientation I think a piece should be in and then other times, I realize that I've painted some of my best work upside down. Also, I have made some pieces that I intended to be displayed in one orientation and then someone else came along to buy it and really insisted that they wanted to hang it in a different orientation. Some artists are more protective of their vision at different times than others for various reasons. I love that you questioned this detail and I'm interested to see if @lizneely has a perspective.

**Comment by lizneely** [comment]
Hello – 
Thanks for the interesting discussion! I certainly didn’t mean to make it sound like I thought you all hadn’t already seen the limitations of only branching out from one authority! It’s just something that has been on my mind a lot and thinking about how cool it would be for authority links to branch to other authorities (and associated information) and to be data-informed and evolving as a living generative graph.

 @ajs6f – We are currently using vocabularies (glue points) on people/organization data and classification/type data. We have place data on our mind, but that is more complex and deserves its own project (connected with our viewshed work with the Land Trust). As you would suspect, people/organization comes up in a variety of categories (maker/creator/designer/manufacturer, depicted, correspondent, owner, exhibitor, exhibition venue, etc.) as does type/classification (type of artwork, genre type in archives). 

For these areas, the following are the authorities (in the case of person in terms of priority, i.e. if there is no ULAN entry use LCNAF, if there is no ULAN nor LCNAF use Wikidata.)

People/Organization:
1/ ULAN
2/LCNAF
3/Wikidata
4/Local Source (and we’ll be working on minimizing these through research. For the ones for which we can find data, we’ll create wikidata/wikipedia entries.(Edit-a-thon!))

Type/Classification:
AAT (behind the scenes we also use Nomenclature for our historic house collections – but will have to address this more when it goes LOD. Nomenclature is not currently serving our LOD – I convert those things to appropriate AAT terms.) In Archives we use AAT for in the Genreform area to distinguish and allow for query on different types of archives (photographs, correspondence, etc.)

Place (future):
TGN

**Comment by lizneely** [comment]
@zeroexp 
 As to your note - I had corresponded with Rob separately before seeing your query, sorry about that.

As for orientation, Georgia O’Keeffe correspondence with Steiglitz reveals she preferred the orientation as portrait and not landscape as he had exhibited it at his gallery 291 (and in the photograph that Rob shared).  (Source: Catalogue Raisonne (CR)) and I believe (though I didn’t dive deeply) that the exhibition installation photo at 291 is also in the CR (attaching a snapshot of the page.) 

These CR conclusions are not all yet input into our data set, but it should be! And then we should link directly to the correspondence (though that could be in our collection or in the Beinecke -- I didn't check and hopefully in the future with linked data it won't matter. :) )
![GOK_Gallery](https://user-images.githubusercontent.com/32772696/57344826-ec047100-7105-11e9-9e9f-093efc7cca62.jpg)
